src/views/InvolvementProfile/components/Membership/components/MemberList/components/MemberListItem/index.js

```diff
@@ -356,4 +406,41 @@ const MemberListItem = ({
   );
 };
 
+MemberListItem.propTypes = {
+  member: PropTypes.shape({
+    MembershipID: PropTypes.number.isRequired,
+    ActivityCode: PropTypes.string.isRequired,
+    //ActivityImage: Not sure what the prop type should be, but it will be required.
```

Comment:
Do we use this? If so, we can find out what the required type is. If not, we can remove it.

Reply:
just deleted them they were not useful great catch @josh-peters-2357

Reply:
Woah woah woah no the ones you just commented out are used all over your code. Activity Image was the one that was commented out originally because it looks like it isn't used. But we needed to find out why it wasn't, and why the bottom three of the props list aren't as well.

Reply:
Better now 😎🎂